make misc/test_blas failed in ubuntu14.04

# Summary
after running configure, failed to make misc/test_blas
I'm sure openblas is installed(apt-get and install from source),and the link flags are correctly set. error info as follows
```
test_blas.cpp:(.text+0x187)：undefined reference to `sgemm_'
test_blas.cpp:(.text+0x346)：undefined reference to `sgeqrf_'
collect2: error: ld returned 1 exit status
```
I have tried solutions from #15  and #131 
<!-- Facebook has a bounty program for the safe disclosure of security bugs. In
those cases, please go through the process outlined on that page and do not
file a public issue. -->

# Platform

OS: <!-- e.g. macOS 10.13.3 -->ubuntu 14.04
Faiss version: <!-- git commit, e.g. 56383610bcb982d6591e2e2bea3516cb7723e04a -->1fe2872013685092d697f08a2a48e110acd25b2b

Faiss compilation options: <!-- e.g. using MKL with compile flags ... -->
```
g++ -std=c++11  -fopenmp  -lopenblas -llapack  -o misc/test_blas misc/test_blas.cpp
#or 
g++ -std=c++11  -fopenmp  -lopenblas -llapack -L/opt/OpenBLAS/lib -I/opt/OpenBLAS/include -o misc/test_blas misc/test_blas.cpp
````


Running on :
- [x] CPU

# Reproduction instructions
<!-- Please provide specific and comprehensive instructions to reproduce the
described behavior. -->

```
./configure
make misc/test_blas.cpp
```
<!-- Please *do not* post screenshots of logs. They are not searchable. Copy/paste 
the text or make a gist if the text is too bulky. --> 
```
test_blas.cpp:(.text+0x187)：undefined reference to `sgemm_'
test_blas.cpp:(.text+0x346)：undefined reference to `sgeqrf_'
collect2: error: ld returned 1 exit status
ldconfig -p |grep openblas
	libopenblas.so.0 (libc6,x86-64) => /usr/local/lib/libopenblas.so.0
	libopenblas.so.0 (libc6,x86-64) => /usr/lib/libopenblas.so.0
	libopenblas.so (libc6,x86-64) => /usr/local/lib/libopenblas.so
	libopenblas.so (libc6,x86-64) => /usr/lib/libopenblas.so
 ldconfig -p |grep lapack
	liblapack.so.3 (libc6,x86-64) => /usr/lib/liblapack.so.3
	liblapack.so (libc6,x86-64) => /usr/lib/liblapack.so
```

--- a/Makefile
+++ b/Makefile
@@ -79,7 +79,7 @@ demos: libfaiss.a
 # Misc
 
 misc/test_blas: misc/test_blas.cpp
-	$(CXX) $(CXXFLAG) $(LDFLAGS) $(LIBS) -o $@ $^
+	$(CXX) $(CXXFLAG) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 
 #############################

